components/automate-cli/cmd/chef-automate/automateHAAwsDeployment.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	"github.com/chef/automate/components/automate-cli/pkg/status"
 	"github.com/chef/automate/components/automate-deployment/pkg/toml"
+	mtoml "github.com/chef/automate/components/automate-deployment/pkg/toml"
```

Comment:
is this not same as above import ?
Please use simply `toml.Marshal(sharedConfigToml)` instead of `mtoml.Marshal(sharedConfigToml)`